experimental/qnamaker-activelearning/csharp_dotnetcore/Helpers/ActiveLearningHelper.cs

```diff
@@ -55,7 +55,7 @@ public static List<QueryResult> GetLowScoreVariation(List<QueryResult> qnaSearch
             var topAnswerScore = qnaSearchResults[0].Score * 100;
             var prevScore = topAnswerScore;
 
-            if ((topAnswerScore > MinimumScoreForLowScoreVariation) && (topAnswerScore < MaximumScoreForLowScoreVariation))
+            if ((topAnswerScore > MinimumScoreForLowScoreVariation) && (topAnswerScore <= MaximumScoreForLowScoreVariation))
```

Comment:
Typically ranges are specified as start to end with the start inclusive and the end exclusive.
Looks like you are doing the opposite here?

Reply:
The range that we want to perform active learning is (20, 95]. It is by design.